src/Command/Shared/FormTrait.php

```diff
@@ -40,9 +40,16 @@ public function formQuestion()
             }
             sort($input_types);
 
+            $this->getIo()->writeln(sprintf(
+              $this->trans('commands.common.messages.available-field-types'), implode(', ', $input_types)
+            ));
+            $this->getIo()->newLine();
+
             $inputs = [];
             $fieldSets = [];
             while (true) {
+                $this->getIo()->info($this->trans('commands.common.questions.inputs.new-field'));
```

Comment:
@LOBsTerr Can we use `comment`  instead of  `info`? 

To show the message in yellow color instead to make clear is not another question?

Reply:
@jmolivas Sure, it is a good idea